### DISCUSSION TITLE
AREA SCAMBIO MATERIALE DOCENTI-ALUNNI

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#100] [Idee e proposte]
Buongiorno,sarebbe possibile creare un'area di scambio materiale didattico tra docenti e alunni?So già che mi direte che esiste classroom, ma avere il tutto all'interno del registro è molto più semplice ed intuitivo.

**Comment by trinko** [comment]
La aggiungo alla lista delle richieste.
